packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -453,4 +442,22 @@ contract ERC20Vault is BaseVault {
             IQuotaManager(quotaManager).consumeQuota(_token, _amount);
         }
     }
+
+    function _safeSymbol(address _token) private view returns (string memory symbol_) {
+        (bool success, bytes memory data) =
+            address(_token).staticcall(abi.encodeCall(IERC20Metadata.symbol, ()));
+        return success ? data.toString() : "";
+    }
+
+    function _safeName(address _token) private view returns (string memory) {
+        (bool success, bytes memory data) =
+            address(_token).staticcall(abi.encodeCall(IERC20Metadata.name, ()));
+        return success ? data.toString() : "";
+    }
```

Comment:
Maybe these 2 (name, symbol) can be put into BaseNFTVault too ?

Reply:
Done